llvm/test/CodeGen/RISCV/GlobalISel/legalizer/rv32/legalize-smax.mir

```diff
@@ -1,5 +1,7 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
 # RUN: llc -mtriple=riscv32 -run-pass=legalizer %s -o - | FileCheck %s
+# RUN: llc -mtriple=riscv32 -mattr=+zbb -run-pass=legalizer %s -o - \
+# RUN:   | FileCheck %s --check-prefix=ZBB
```

Comment:
What do you think about adding BOTH,NOZBB and BOTH,ZBB 
lines instead of duplicating check lines? I think in this test some of the functions in this test/PR will only use BOTH.

Reply:
There should be no duplicates.

Reply:
Nevermind, the very last test is duplicate because of the illegal type.